SimCalorimetry/HGCalSimProducers/interface/HGCFEElectronics.h

```diff
@@ -43,13 +43,27 @@ class HGCFEElectronics
       }
   }
 
+
+  void SetNoiseValues(std::vector<float> noise_fC){
+    for( auto noise : noise_fC ) { noise_fC_.push_back( noise ); }
+  };
+
+  float getTimeJitter(float totalCharge, int thickness){
+    float A2 = jitterNoise2_ns_[thickness-1];
```

Comment:
is it *guaranteed* that the value of `thickness` will not exceed the dimension of the vector? If not, I would at least use `.at(thickness-1)` to ensure you never read from uninitialized memory. (& for the two below)